### PR TITLE
Download of libui binaries from andlabs main repo 

### PR DESCRIPTION
**Pull request by parro-it**
In this PR I changed the download process of the binaries to use the binaries attached to release in andlabs repo.

This is a first step in order to implements binding for the new table APIs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
   },
-  "libui": "alpha3.5-master-003",
+  "libui": "alpha4.1",
   "scripts": {
     "docs": "node tools/docs.js",
     "start": "node --expose-gc example.js",
     "download-libui": "node tools/libui-download.js",
     "download-libui-napi": "node tools/libui-napi-download.js",
-    "clean": "rimraf build *.h *.node libui.* node_modules",
+    "clean": "rimraf build *.h *.node libui.* node_modules/!(rimraf|.bin)",
```
Comment (mischnic) [inline package.json]: Why does this delete the node_modules folder?

**Comment by mischnic** [comment]
Travis fails on macOS because of the text background issue on High Sierra, Linux on Node 6 is probably just a false negative and AppVeyor didn't post the results to now.

File: package.json (hunk 1 above)
Comment (parro-it) [reply package.json]: I had an issue on Windows: executing rimraf locks it's binary file, so WIndows is not able to delete it. If we don't exclude it, the scripts fails with an error.

**Comment by parro-it** [comment]
Ok, so I think I can merge this. 
Meanwhile, I'm working on table binding in 'tables' branch that is based on this one.

File: package.json (hunk 1 above)
Comment (mischnic) [reply package.json]: My question was: Why delete node_modules at all?
Is `clean` meant as the counterpart to `npm run install` or `npm install`?

Comment (mischnic) [reply package.json]: > What's the difference between the two?

`npm install`: Install dependencies and run the `install` script from package.json
`npm run install`: Only run the `install` script from package.json

Comment (parro-it) [reply package.json]: Oh I didn't know that! Nice!

**Comment by mischnic** [comment]
> Ok, so I think I can merge this.

... ?

**Comment by parro-it** [comment]
ups, forgotten

**Pull request merged**